PublicGitArchive/pga/cmd/get.go

```diff
@@ -17,6 +17,8 @@ import (
 	pb "gopkg.in/cheggaaa/pb.v1"
 )
 
+const rootURL = "http://pga.sourced.tech/"
```

Comment:
Wow I can't believe this was hardcoded before. Shame

Reply:
haha :p I guess nobody wanted to mess with Francesc's code :-1: